package.json

```diff
@@ -48,6 +48,7 @@
   },
   "dependencies": {
     "debug": "^2.2.0",
+    "electron-editor-context-menu": "^1.1.0",
```

Comment:
Can we just pull the code into the desktop repo as a new module instead of adding as a dependency (especially since it's just one small file)? With proper credit + links, of course.